content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformationcleanup.yaml

```diff
@@ -1,37 +0,0 @@
----
```

Comment:
@jameswnl why do you remove this state machine ? It's called when migration fails to ensure that we return to initial state. But it's not always done...

Reply:
Good question, please review [here](https://github.com/ManageIQ/manageiq/pull/18415/files#diff-51b91017cbf3d1648a52ca0482a63650R58)

Reply:
Reviewed, but I don't see where cancelation happens. Could you point to the methods that effectively kill virt-v2v and power the source VM on when the migration fails, not only on cancellation ?

Reply:
How was this cleanup invoked previously?